The restricted attribute list will prevent

dynamic-entropy opened the pull request:
The restricted attribute list will prevent users (including admins and operators) from modifying unwanted attributes.

For e.g. We want to prevent archive monitoring step performed by FTS on MIT_Tape; this requires that `archive_timeout` is not set on the RSE.

Fixes https://github.com/dmwm/CMSRucio/issues/726

ericvaandering commented:
The PR description is not quite complete I think. Can you fill it in? Then we can merge.

dynamic-entropy commented:
Hello @ericvaandering, 
I have modified the description and the title. 
The previous title and description were automatically generated by Git Hub from the commit message. It was complete I think, but was split. I have corrected that and have added an example to describe more of what this is.
